build.gradle

```diff
@@ -67,6 +75,22 @@ subprojects {
         configDirectory = rootProject.file("checkstyle/")
     }
 
+    pitest {
+        //adds dependency to org.pitest:pitest-junit5-plugin and sets "testPlugin" to "junit5"
+        junit5PluginVersion = '1.1.0'
+        pitestVersion = '1.14.1'
+        targetClasses.add("io.aiven.kafka.tieredstorage.*")  //by default "${project.group}.*"
+        targetTests.add("io.aiven.kafka.tieredstorage.*")  //by default "${project.group}.*"
+        threads = 4
```

Comment:
Any reason for this to be a constant and not e.g. `Runtime.getRuntime().availableProcessors()`?

Reply:
No, no reason, applied suggestion.